content/opt/setup/graphite

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Graphite writer
+if [ "${ICINGA2_FEATURE_GRAPHITE}" == "true" ] || [ "${ICINGA2_FEATURE_GRAPHITE}" == "1" ]; then
+	echo "=> Enabling Icinga2 graphite writer"
+
+	icinga2 feature enable graphite
+
+	cat >/etc/icinga2/features-available/graphite.conf <<-END
+	/**
+	 * The GraphiteWriter type writes check result metrics and
+	 * performance data to a graphite tcp socket.
+	 */
+	library "perfdata"
+	object GraphiteWriter "graphite" {
+	  host = "$ICINGA2_FEATURE_GRAPHITE_HOST"
+	  port = "$ICINGA2_FEATURE_GRAPHITE_PORT"
+	  enable_send_thresholds = true
+	}
+	END
+
+else
+	# Actively disable graphite, to not hit any weird bugs
+	icinga2 feature disable graphite
```

Comment:
No need to disable graphite as it is not enabled by default, right?

The container output might be a bit confusing for users:
```
critical/cli: Cannot disable feature 'graphite'. Target file '/etc/icinga2/features-enabled/graphite.conf' does not exist.
critical/cli: Cannot disable feature(s): graphite
=> Copying fresh config-files for /etc/icingaweb2
/opt/setup/icingaweb2: line 31: ini: command not found
ERROR: Icinga\Exception\ConfigurationError in /usr/share/php/Icinga/Application/Modules/Manager.php:318 with message: Cannot disable module "graphite". Module is not installed.
```